.changeset/green-icons-jog.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/lab": minor
+---
+
+Added `validationStatus` to InputNext
+Added `StatusAdornment`, `ErrorAdornment` and `WarningAdornment` components
```

Comment:
do we want these in a lab/internal folder? @joshwooding

Reply:
changelog needs to be updated

Reply:
@alycrys updated

Reply:
> do we want these in a lab/src/internal folder? @joshwooding

For now I'll say no, consumers might want to use them in their custom components